Documentation/monitoring-kubernetes.md

```diff
@@ -0,0 +1,27 @@
+# Monitoring Kubernetes
```

Comment:
Maybe we should concentrate on how to run exporters in this PR as an elaborate guide for cluster monitoring is also here https://github.com/coreos/prometheus-operator/pull/188. I will make sure to include things from here in that cluster monitoring guide that are missing.